cfme/tests/cloud/test_providers.py

```diff
@@ -82,7 +82,7 @@ def test_provider_crud(provider):
         test_flag: crud
     """
     provider.create()
-    provider.validate(db=False)
+    provider.validate(ui=True)
```

Comment:
The only way I can think of solving this is if varmeth uses the default if the requested isn't present and logs?